### ISSUE TITLE
RTE BOW SuperGLUE baseline broken

### ISSUE THREAD

**Issue by sleepinyourhat** [nyu-mll/jiant#822]
Running the standard SuperGLUE RTE BOW baseline command from head, in a fresh directory with a fresh download, yields the following:

```
07/08 05:05:38 PM: Saved config to /Users/Bowman/Drive/JSALT/bow-rte-v2/rte/params.conf
07/08 05:05:38 PM: Using random seed 1234
07/08 05:05:38 PM: GPU access failed. You might be using a CPU-only installation of PyTorch. Falling back to CPU.
07/08 05:05:38 PM: Loading tasks...
07/08 05:05:38 PM: Writing pre-preprocessed tasks to /Users/Bowman/Drive/JSALT/bow-rte-v2/
07/08 05:05:38 PM: 	Creating task rte-superglue from scratch.
07/08 05:05:38 PM: 	Loading Tokenizer MosesTokenizer
07/08 05:05:41 PM: 	Finished loading RTE (from SuperGLUE formatted data).
07/08 05:05:41 PM: 	Task 'rte-superglue': |train|=2490 |val|=277 |test|=3000
07/08 05:05:41 PM: 	Finished loading tasks: rte-superglue.
07/08 05:05:41 PM: 	Building vocab from scratch.
07/08 05:05:41 PM: 	Counting words for task rte-superglue.
07/08 05:05:42 PM: 	Saved vocab to /Users/Bowman/Drive/JSALT/bow-rte-v2/vocab
07/08 05:05:42 PM: Loading token dictionary from /Users/Bowman/Drive/JSALT/bow-rte-v2/vocab.
07/08 05:05:42 PM: 	Loaded vocab from /Users/Bowman/Drive/JSALT/bow-rte-v2/vocab
07/08 05:05:42 PM: 	Vocab namespace chars: size 108
07/08 05:05:42 PM: 	Vocab namespace tokens: size 18927
07/08 05:05:42 PM: 	Finished building vocab.
07/08 05:05:42 PM: 	Building embeddings from scratch.
07/08 05:05:58 PM: 	Finished loading embeddings
07/08 05:05:58 PM: 	Saved embeddings to /Users/Bowman/Drive/JSALT/bow-rte-v2/embs.pkl
07/08 05:05:58 PM: Trimmed word embeddings: torch.Size([18927, 300])
07/08 05:05:58 PM: 	Task rte-superglue (train): Indexing from scratch.
07/08 05:05:58 PM: 	Task rte-superglue (train): Saved 0 instances to /Users/Bowman/Drive/JSALT/bow-rte-v2/preproc/rte-superglue__train_data
07/08 05:05:58 PM: 	Task rte-superglue (val): Indexing from scratch.
07/08 05:05:58 PM: 	Task rte-superglue (val): Saved 0 instances to /Users/Bowman/Drive/JSALT/bow-rte-v2/preproc/rte-superglue__val_data
07/08 05:05:58 PM: 	Task rte-superglue (test): Indexing from scratch.
07/08 05:05:58 PM: 	Task rte-superglue (test): Saved 0 instances to /Users/Bowman/Drive/JSALT/bow-rte-v2/preproc/rte-superglue__test_data
07/08 05:05:58 PM: 	Finished indexing tasks
07/08 05:05:58 PM: 	Creating trimmed pretraining-only version of rte-superglue train.
07/08 05:05:58 PM: 	Creating trimmed target-only version of rte-superglue train.
07/08 05:05:58 PM: 	  Training on rte-superglue
07/08 05:05:58 PM: 	  Evaluating on rte-superglue
07/08 05:05:58 PM: 	Finished loading tasks in 19.705s
07/08 05:05:58 PM: 	 Tasks: ['rte-superglue']
07/08 05:05:58 PM: Building model...
07/08 05:05:58 PM: 	Using pre-trained word embeddings: torch.Size([18927, 300])
07/08 05:05:58 PM: 	Not using character embeddings!
07/08 05:05:58 PM: Initializing parameters
07/08 05:05:58 PM: Done initializing parameters; the following parameters are using their default initialization from their code
07/08 05:05:58 PM:    _text_field_embedder.token_embedder_words.weight
07/08 05:05:58 PM: 	Task 'rte-superglue' params: {
  "cls_type": "log_reg",
  "d_hid": 512,
  "pool_type": "max",
  "d_proj": 512,
  "shared_pair_attn": 0,
  "attn": 0,
  "d_hid_attn": 512,
  "dropout": 0.2,
  "cls_loss_fn": "softmax",
  "cls_span_pooling": "attn",
  "edgeprobe_cnn_context": 0,
  "use_classifier": "rte-superglue"
}
07/08 05:05:58 PM: Model specification:
07/08 05:05:58 PM: MultiTaskModel(
  (sent_encoder): BoWSentEncoder(
    (_text_field_embedder): ElmoTextFieldEmbedder(
      (token_embedder_words): Embedding()
    )
  )
  (rte-superglue_mdl): PairClassifier(
    (pooler): Pooler(
      (project): Linear(in_features=300, out_features=512, bias=True)
    )
    (classifier): Classifier(
      (classifier): Linear(in_features=2048, out_features=2, bias=True)
    )
  )
)
07/08 05:05:58 PM: Model parameters:
07/08 05:05:58 PM: 	sent_encoder._text_field_embedder.token_embedder_words.weight: Non-trainable parameter, count 5678100 with torch.Size([18927, 300])
07/08 05:05:58 PM: 	rte-superglue_mdl.pooler.project.weight: Trainable parameter, count 153600 with torch.Size([512, 300])
07/08 05:05:58 PM: 	rte-superglue_mdl.pooler.project.bias: Trainable parameter, count 512 with torch.Size([512])
07/08 05:05:58 PM: 	rte-superglue_mdl.classifier.classifier.weight: Trainable parameter, count 4096 with torch.Size([2, 2048])
07/08 05:05:58 PM: 	rte-superglue_mdl.classifier.classifier.bias: Trainable parameter, count 2 with torch.Size([2])
07/08 05:05:58 PM: Total number of parameters: 5836310 (5.83631e+06)
07/08 05:05:58 PM: Number of trainable parameters: 158210 (158210)
07/08 05:05:58 PM: Finished building model in 0.012s
07/08 05:05:58 PM: Will run the following steps for this experiment:
Training model on tasks: rte-superglue 
Evaluating model on tasks: rte-superglue 

07/08 05:05:58 PM: Training...
07/08 05:05:58 PM: patience = 20
07/08 05:05:58 PM: val_interval = 625
07/08 05:05:58 PM: max_vals = 10000
07/08 05:05:58 PM: cuda_device = -1
07/08 05:05:58 PM: grad_norm = 5.0
07/08 05:05:58 PM: grad_clipping = None
07/08 05:05:58 PM: lr_decay = 0.99
07/08 05:05:58 PM: min_lr = 1e-07
07/08 05:05:58 PM: keep_all_checkpoints = 0
07/08 05:05:58 PM: val_data_limit = 5000
07/08 05:05:58 PM: max_epochs = 10
07/08 05:05:58 PM: dec_val_scale = 250
07/08 05:05:58 PM: training_data_fraction = 1
07/08 05:05:58 PM: Fatal error in main():
Traceback (most recent call last):
  File "main.py", line 576, in <module>
    main(sys.argv[1:])
  File "main.py", line 507, in main
    phase="pretrain",
  File "/Users/Bowman/Drive/JSALT/jiant/jiant/trainer.py", line 486, in train
    tasks, batch_size, train_params, optimizer_params, scheduler_params, phase
  File "/Users/Bowman/Drive/JSALT/jiant/jiant/trainer.py", line 315, in _setup_training
    instance = [i for i in itertools.islice(task.train_data, 1)][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "main.py", line 587, in <module>
    raise e  # re-raise exception, in case debugger is attached.
  File "main.py", line 576, in <module>
    main(sys.argv[1:])
  File "main.py", line 507, in main
    phase="pretrain",
  File "/Users/Bowman/Drive/JSALT/jiant/jiant/trainer.py", line 486, in train
    tasks, batch_size, train_params, optimizer_params, scheduler_params, phase
  File "/Users/Bowman/Drive/JSALT/jiant/jiant/trainer.py", line 315, in _setup_training
    instance = [i for i in itertools.islice(task.train_data, 1)][0]
IndexError: list index out of range
```

**Comment by sleepinyourhat** [comment]
(Not model-specific, this is just the simplest case I could find to reproduce it. On a Mac with a current Conda setup.)

**Comment by pruksmhc** [comment]
I'll be able to get to this in about an hour, but if you want to get a fix for this - this is because superglue-diagnostics was moved to RTE/diagnostics subdirectory, so you'll have to change the rel-path of superglue-diagnostic task.

**Comment by W4ngatang** [comment]
Resolved with #823
